### PR TITLE
Dump LAr density in DetectorProperties unit test

### PR DESCRIPTION
**Pull request by PetrilloAtWork**

### CODE REVIEW

File: test/DetectorInfo/DetectorPropertiesStandard_test.cc [hunk 1]
```diff
@@ -123,6 +123,8 @@ main(int argc, char const** argv)
                                << " kV/cm"
                                << "\nSampling rate:       " << TDCtick << " ns"
                                << "\nArgon temperature:   " << detp.Temperature() << " K"
+                               << "\nArgon density:       " << detp.Density() << " kg/dm^3 (at " << detp.Temperature() << " K)"
+                               << "\nArgon density:       " << detp.Density(83.0) << " kg/dm^3 (at 87 K)"
```
Comment (knoepfel) [inline test/DetectorInfo/DetectorPropertiesStandard_test.cc]: Should this be `detp.Density(87.0)`?

**Comment by knoepfel** [comment]
code-checks

File: test/DetectorInfo/DetectorPropertiesStandard_test.cc (hunk 1 above)
Comment (PetrilloAtWork) [reply test/DetectorInfo/DetectorPropertiesStandard_test.cc]: It should have.
I have tried editing on GitHub... let's see if I managed.

**Comment by lgarren** [comment]
trigger build

**Comment by lgarren** [comment]
trigger build

**Comment by lgarren** [comment]
trigger build

**Comment by lgarren** [comment]
approve

**Pull request merged**